gipc/gipc.py

```diff
@@ -1058,6 +1061,33 @@ def put(self, o):
             self._write(struct.pack("!i", len(bindata)) + bindata)
 
 
+if not WINDOWS and FORK_MODE == 'spawn':
```

Comment:
Do you maybe want to add a comment here pointing to this PR discussion, and maybe mentioning that this is the default since CPython 3.8 on macOS?